src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -329,7 +329,20 @@ const PIAFormPage = () => {
               ? PiaStatuses.MPO_REVIEW
               : PiaStatuses.INCOMPLETE,
         });
-        if (pia?.id) {
+        if (
+          (pia?.id &&
+            pia?.isNextStepsSeenForNonDelegatedFlow === true &&
+            pia?.hasAddedPiToDataElements === true) ||
```

Comment:
true or null [for not sure] ??

```suggestion
          (pia?.id &&
            pia?.isNextStepsSeenForNonDelegatedFlow === true &&
            (pia?.hasAddedPiToDataElements === true || pia?.hasAddedPiToDataElements === null) ||
```

Reply:
Should we use the const PI options for consistency and clarity when evaluating this:
export const PIOptions = [
  {
    key: 'Yes',
    value: true,
  },
  {
    key: 'No',
    value: false,
  },
  {
    key: "I'm not sure",
    value: null,
  },
];

Reply:
Shouldn't it never be null? As a radio value, it will always have a default value.

Reply:
@antsand I feel as though drilling into an array of objects for a boolean evaluation is more confusing that simply checking for the boolean. This may seem like a departure from our convention (i.e.: YesNoInput), but in those cases we're checking for string values where capitalization / spelling matter.